src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import DrawingAnnotations from '../DrawingAnnotations';
+import DrawingList from '../DrawingList';
+import { annotations } from '../__mocks__/drawingData';
+
+jest.mock('../DrawingList');
+
+describe('DrawingAnnotations', () => {
+    const defaults = {
+        activeAnnotationId: null,
+        annotations: [],
+        isCurrentFileVersion: true,
+        setActiveAnnotationId: jest.fn(),
+    };
+    const getWrapper = (props = {}): ShallowWrapper => shallow(<DrawingAnnotations {...defaults} {...props} />);
+
+    describe('event handlers', () => {
+        describe('handleAnnotationActive()', () => {
+            test('should call setActiveAnnotationId with annotation id', () => {
+                getWrapper()
+                    .find(DrawingList)
+                    .prop('onSelect')!('123');
+
+                expect(defaults.setActiveAnnotationId).toHaveBeenCalledWith('123');
+            });
+        });
+    });
+
+    describe('render()', () => {
+        test('should render one DrawingAnnotation per annotation', () => {
+            const wrapper = getWrapper({ annotations });
+            const list = wrapper.find(DrawingList);
+
+            expect(list.hasClass('ba-DrawingAnnotations-list')).toBe(true);
+            expect(list.prop('annotations').length).toBe(annotations.length);
```

Comment:
This doesn't really assert that `DrawingAnnotation`s are being rendered due to it being shallow rendered which the test name seems to suggest, right?

Reply:
Yeah, I actually just want to test if the rendered list has the right props. Updated.